preferences/rhel/10/amd64/kustomization.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../../base
+
+components:
+  - ./metadata
+  - ./requirements
+  - ../../../components/disk-dedicatediothread
+  - ../../../components/rng
+  - ../../../components/secureboot
```

Comment:
@0xFelix can you please disable secure boot in the initial version?

Reply:
Addressed